arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/StackFragment.java

```diff
@@ -75,12 +75,14 @@ private StackFragment(
     this.exceptions = exceptions;
     this.opCode = stack.getCurrentOpcodeData().mnemonic();
     this.hashInfoFlag =
-        switch (this.opCode) {
-          case SHA3 -> Exceptions.none(exceptions) && gp.messageSize() > 0;
-          case RETURN -> Exceptions.none(exceptions) && gp.messageSize() > 0 && isDeploying;
-          case CREATE2 -> Exceptions.none(exceptions) && aborts.none() && gp.messageSize() > 0;
-          default -> false;
-        };
+        Exceptions.none(exceptions)
```

Comment:
I exchanged positions so that these checks are done after the `switch`. I believe that Java is smart enough to ignore conditions in a chain of `&&`'s if a previous condition was false.